security/network_bound_disk_encryption/nbde-tang-server-installation-considerations.adoc

```diff
@@ -0,0 +1,24 @@
+// CNF-2127 assembly
+[id="nbde-tang-server-installation-considerations"]
+= Tang server installation considerations
+include::modules/common-attributes.adoc[]
+:context: nbde-implementation
+
+toc::[]
+
+include::modules/nbde-installing-a-tang-server.adoc[leveloffset=+1]
+
+include::modules/nbde-compute-requirements.adoc[leveloffset=+2]
```

Comment:
My $0.02 is that the conceptual information from the modules on lines 11 to 15 don't have a lot to do with the installation process.  See if there is a topic in the "About" page where they fit better.

Conversely, try putting "Installation scenarios" ahead of the first include so that customers are given a few considerations for planning and then read the installation procedure.

Reply:
Moved  "Installation scenarios" ahead of the first include